src/mimefactory.rs

```diff
@@ -593,6 +593,15 @@ impl<'a> MimeFactory<'a> {
             ));
         }
 
+        if let Loaded::Message { chat } = &self.loaded {
+            if chat.typ == Chattype::Broadcast {
+                headers.unprotected.push(Header::new(
```

Comment:
Can we add "List-ID" to protected headers instead, or will the current code not parse it correctly?

Reply:
So, for now it's not a problem because broadcast messages are not encrypted anyway